packages/fluent-theme/src/components/assets/AssetComposer.tsx

```diff
@@ -0,0 +1,37 @@
+
+const AssetComposer = memo(({ children }: AssetComposerProps) => {
+  const slidingDotsURL = useMemo(
+    () => URL.createObjectURL(new Blob([SLIDING_DOTS_SVG_STRING], { type: 'image/svg+xml' })),
+    []
+  );
+
+  useEffect(() => () => URL.revokeObjectURL(slidingDotsURL), [slidingDotsURL]);
+
+  const context = useMemo<ContextType>(
+    () =>
+      Object.freeze({
+        urlStateMap: new Map<AssetName, readonly [URL]>([['sliding dots', Object.freeze([new URL(slidingDotsURL)])]])
```

Comment:
Maybe leverage the outside context if any? But waits for the future work. Seems we found perfect way to share assets which can be replaced in both JS and CSS :sunglasses:

Reply:
Oh right, should do this... later.